core/src/main/java/io/aiven/klaw/controller/Cache.java

```diff
@@ -0,0 +1,43 @@
+package io.aiven.klaw.controller;
+
+import io.aiven.klaw.config.ManageDatabase;
+import io.aiven.klaw.dao.Env;
+import io.aiven.klaw.model.ApiResponse;
+import io.aiven.klaw.model.enums.*;
+import jakarta.validation.Valid;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/cache")
+@Slf4j
+public class Cache {
+
+  @Autowired ManageDatabase manageDatabase;
+
+  @PostMapping(
+      value = "/environment/tenant/{tenantId}/id/{id}",
+      produces = {MediaType.APPLICATION_JSON_VALUE},
+      consumes = {MediaType.APPLICATION_JSON_VALUE})
+  public ResponseEntity<ApiResponse> addEnvToCache(
+      @PathVariable("tenantId") Integer tenantId,
+      @PathVariable("id") Integer id,
```

Comment:
if this is environmentId, it could be better to rename both these variables to envId ?

Reply:
I think id is sufficient here as it is the id of the "/environment/" however I am ok with changing this if you prefer it to be more explicit.

Reply:
No,I think it's ok.